results.tex

```diff
@@ -229,8 +235,8 @@ \subsection{Regional Tariffs: Fuel Cycles with International Instruments}
 Two regions, Region A and Region B, are modeled. Region A houses a fuel cycle
 with both UOX and MOX-based fuel services, as in the \basecase scenario. The
 same total number of \reactors are modeled in the scenario. Region A begins with
-15 \reactors and Region B begins with 5 reactors. All reactor deployment occurs
-in Region A as shown in Figure \ref{fig:deploy}.
+15 \reactors and Region B begins with 5 reactors. All reactor deployment as
+shown in Figure \ref{fig:deploy} occurs in Region A.
```

Comment:
I'm not sure I agree with this suggested change.   I find the new version to be awkward.